resources/user-agents/browsers/maxthon/maxthon-4-1-to-4-3.json

```diff
@@ -380,20 +369,44 @@
             "Win8_1_x64", "Win8_1_64", "Win8_1_32",
             "Win10_x64", "Win10_64", "Win10_32", "Win10_x64_B", "Win10_64_B", "Win10_32_B", "Windows"
           ]
+        }
+      ]
+    },
+    {
+      "userAgent": "Maxthon #MAJORVER#.#MINORVER# for Mac",
```

Comment:
This was something I wanted to test.  By having the default platform/device be similar to what the children are, a lot of lines can be saved when there are a lot of patterns in the UA group. For browsers like this one, it may make sense to split up the desktop patterns into their own UA group (Linux, Mac, Windows).

The biggest downside to this that I see is that we have to duplicate the browser properties in each UA group, but with the browser extraction that @mimmi20 was thinking about doing, there would be a lot less of that, which may make this a more attractive approach to keep the final INI file size smaller.

Thoughts?

For this PR, I only did this for this file and the next two maxthon files.